src/ape/api/address.py

```diff
@@ -90,6 +90,15 @@ def __str__(self) -> str:
         """
         return self.address
 
+    def __hash__(self) -> int:
+        """Return consistent hash for all address representations with the same value."""
+        value = self.address
```

Comment:
`self.address` is already defined as `AddressType`, so don't think we need the extra conversion

Reply:
Thanks. Fixed it!